src/Input.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components'
+import { space } from 'styled-system'
+
+const Input = styled.input`
+  appearance: none;
+  display: block;
+  width: 100%;
+  font-family: inherit;
+  font-size: ${({ theme }) => theme.fontSizes[2]}px;
```

Comment:
Not necessary to use here, but styled-system has a [`theme`](https://github.com/jxnblk/styled-system#theme) getter utility to pick up values like this with `${styledSystem.theme('fontSizes.2')}px`